docs/guide/imitation.rst

```diff
@@ -0,0 +1,55 @@
+.. _imitation:
+==================
+Imitation Learning
+==================
+
+The `imitation <https://github.com/HumanCompatibleAI/imitation>`_ library implements
+imitation learning algorithms on top of Stable-Baselines3, including:
+
+  - Behavioral Cloning
+  - `DAgger <https://arxiv.org/pdf/1011.0686.pdf>`_ with synthetic examples
+  - `Adversarial Inverse Reinforcement Learning <https://arxiv.org/abs/1710.11248>`_ (AIRL)
+  - `Generative Adversarial Imitation Learning <https://arxiv.org/abs/1606.03476>`_  (GAIL)
+
+
+It also provides `CLI scripts <#Sacred-CLI-Quickstart>`_ for training and saving
+demonstrations from RL experts, and for training imitation learners on these demonstrations.
+
+
+Installation
+------------
+
+Requires Python 3.7+.
```

Comment:
Why is that? What are you using which make it incompatible with python 3.6?

Reply:
`dataclasses` is one thing (used as a data type for transitions and trajectories).

Reply:
that's a nice feature =) (I did not know about, I've some hack to do that before)